File: 0027-remove-element/0027-remove-element.py
class Solution:
    def removeElement(self, nums: List[int], val: int) -> int:
        n = len(nums)
        index = 0
        for i in range(n):
            if nums[i] != val:
                nums[index] = nums[i]
                index = index + 1
        return index

File: 0027-remove-element/0027-remove-element.cpp
class Solution {
public:
    int removeElement(vector<int>& nums, int val) {
        int count = 0;
        for(int num:nums){
            if(num != val){
                nums[count] = num;
                count ++;
            }
        }
        return count;
        
    }
};